### PR TITLE
Honda Remote

### PR DESCRIPTION
**Pull request by ukshady**
New decoder for honda car remote. Data samples already exist in rtl_433_tests

### CODE REVIEW

File: src/devices/hondaremote.c [hunk 1]
```diff
@@ -0,0 +1,75 @@
+/* Honda Car Key
+ *
+ * Identifies event, but does not attempt to decrypt rolling code...
+ *
+ * Copyright (C) 2015 Tommy Vestermark
```
Comment (merbanan) [inline src/devices/hondaremote.c]: Well you have to add your name here.

File: src/devices/hondaremote.c [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+	bitrow_t *bb = bitbuffer->bb;
+        uint8_t *bytes = bitbuffer->bb[0];
+	char time_str[LOCAL_TIME_BUFLEN];
+	data_t *data;
+
+
+	// Validate package
+if ((bytes[0] ==0xFF ) && (bytes[38] == 0xFF)) 
+	 {
+	fprintf(stdout, "Honda Car Key:\n");
```
Comment (merbanan) [inline src/devices/hondaremote.c]: No fprintf's without debug guards.

File: src/devices/hondaremote.c [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+static const char* get_command_codes(const uint8_t* bytes) {
+	return command_code[bytes[46] - 0xAA];
+	}
+
+static int hondaremote_callback(bitbuffer_t *bitbuffer) {
+	bitrow_t *bb = bitbuffer->bb;
+        uint8_t *bytes = bitbuffer->bb[0];
+	char time_str[LOCAL_TIME_BUFLEN];
+	data_t *data;
+
```
Comment (merbanan) [inline src/devices/hondaremote.c]: Add an early termination check here. Test for more then 300 bits in the row or something like that.

**Comment by ukshady** [comment]
I have made some changes as suggested.

**Comment by merbanan** [comment]
Jolly good.

**Pull request merged**